frontend/src/scenes/notebooks/Notebook/NotebookColumnLeft.tsx

```diff
@@ -30,6 +30,41 @@ export const NotebookColumnLeft = (): JSX.Element | null => {
     )
 }
 
+export const NotebookNodeSettingsOffset = ({ logic }: { logic: BuiltLogic<notebookNodeLogicType> }): JSX.Element => {
+    const { ref } = useValues(logic)
+    const offsetRef = useRef<HTMLDivElement>(null)
+    const [height, setHeight] = useState(0)
+
+    useEffect(() => {
+        // Interval to check the relative positions of the node and the offset div
+        // updating the height so that it always is inline
+        const updateHeight = (): void => {
+            if (ref && offsetRef.current) {
+                const newHeight = ref.getBoundingClientRect().top - offsetRef.current.getBoundingClientRect().top
+
+                if (height !== newHeight) {
+                    setHeight(newHeight)
+                }
+            }
+        }
+
+        const interval = setInterval(updateHeight, 100)
```

Comment:
Noticed that changing the height will causing a new interval to be created each time.
![intervals](https://github.com/PostHog/posthog/assets/6685876/2350b946-7619-4d4c-9872-a45843c994e8)

I would have thought this meant the `updateHeight` would then be called once per interval every 100ms but that doesn't seem to be the case 🤔 Perhaps they are being cleared somehow when unallocated

Reply:
It clears it in the return statement of the effect.

Reply:
I thought the returned function only ran when the component was unmounted. I'm guessing this is a TIL moment and it actually runs whenever the computing properties change